packages/smart-wallet/test/swingsetTests/upgradeWalletFactory/bootstrap-walletFactory-service-upgrade.js

```diff
@@ -84,11 +104,35 @@ export const buildRootObject = () => {
       const currentStoragePath = await E.get(
         E.get(E(wallet).getPublicTopics()).current,
       ).storagePath;
-      currentStoragePath === walletPath || Fail`bad storage path`;
+      currentStoragePath === currentPath || Fail`bad storage path`;
+
+      addAsset('umoola', 'moola', 'moola', moolaKit);
+      const depositFacet = E(wallet).getDepositFacet();
+      const payment = moolaKit.mint.mintPayment(
+        AmountMath.make(moolaKit.brand, 100n),
+      );
+      // @ts-expect-error casting far for test
+      await E(depositFacet).receive(payment);
 
       return true;
     },
 
+    testOfferV1: async () => {
+      // If this doesn't throw, the offer succeeded.
+      // I tried to also check vstorage but it doesn't always update in time.
```

Comment:
yeah... lack of a return promise from `setValue` makes coordinating that sort of thing nearly impossible.